extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -473,21 +418,11 @@ export class ConsentPolicyInstance {
   }
 
   /**
-   * Returns the current consent string info
-   * @return {?string}
-   */
-  getConsentString() {
-    // Return a string because we only support one consent instance now.
-    // And decide not to support multiple consent instances given CMP approach
-    return this.consentString_;
-  }
-
-  /**
-   * Returns whether the current consent policy state should block
+   * Returns whether the current consent policy state should be unblocked
    * Caller need to make sure that policy status has resolved
    * @return {boolean}
    */
-  shouldBlock() {
+  shouldUnblock() {
```

Comment:
I noticed we went from block, to unblock around the new refactored code. What is the reasoning for this? Usually we want to [avoid negative language in conditionals](https://schneide.blog/2014/08/03/dont-ever-not-avoid-negative-logic/). 😄

Reply:
Maybe we can rename "unblock" to enable?

Reply:
LOL 😄 I completely agree with the ariticle!

I changed the name because the value returned is whether go ahead and unblock the policy. It was an incorrect name before. 

I could change it to `shouldEnable`. However we also have the `unblockOn` field from the config, I was thinking `shouldUnblock` would match that name better. Let me know what do you think. Since `unblockOn` is a private field, I am find with renaming that as well.

Reply:
I think we should rename all unblock to enable 😄 Glad you liked the article I dug up haha!